fuel-vm/src/tests/encoding.rs

```diff
@@ -395,4 +410,114 @@ fn transaction() {
             vec![],
         ),
     ]);
+    assert_transactions_encoding_correct(&[
```

Comment:
It's kinda hard to see what is different about each tx or why there are the number there are. Maybe move to a helper that says what the set is? I can't say how you chose them, but whatever the process was can be the name of the helper.

Reply:
https://github.com/FuelLabs/fuel-vm/issues/717